src/frontend/components/actions/new.tsx

```diff
@@ -15,25 +17,30 @@ import LayoutElementRenderer from './utils/layout-element-renderer.js'
 
 const New: FC<ActionProps> = (props) => {
   const { record: initialRecord, resource, action } = props
-  const {
-    record,
-    handleChange,
-    submit: handleSubmit,
-    loading,
-    setRecord,
-  } = useRecord(initialRecord, resource.id)
+  const { record, handleChange, submit, loading, setRecord } = useRecord(initialRecord, resource.id)
   const { translateButton } = useTranslation()
   const navigate = useNavigate()
+  const { parsedQuery } = useQueryParams()
 
   useEffect(() => {
     if (initialRecord) {
       setRecord(initialRecord)
     }
-  }, [initialRecord])
+  }, [initialRecord, parsedQuery])
 
-  const submit = (event: React.FormEvent<HTMLFormElement>): boolean => {
+  useEffect(() => {
+    if (parsedQuery) {
+      const resourceProperties = pick(parsedQuery, Object.keys(resource.properties))
+      if (Object.keys(resourceProperties).length) {
+        setRecord({ ...record, params: { ...record.params, ...resourceProperties } })
```

Comment:
What will happen if we add for example:

`?gender=abcd&isActive=false`?

If `gender` is `male`/`female`/`other` enum, will it set `abcd`?

If `isActive` is a boolean, will it correctly parse `true`/`false` strings?

Reply:
Checked with enums and booleans and there are results

**for enums:**
value in url has to be exact like availableValues e.g.
```
export enum CountryEnum {
  Poland = 'PL',
  Germany = 'DE',
}
```
should look like `?country=PL` (case sensitive). Otherwise value would not be filled in new item form. 
![image](https://user-images.githubusercontent.com/85543813/233103873-babeb7f3-8723-443f-8de7-215a990ce181.png)
![image](https://user-images.githubusercontent.com/85543813/233103906-96fc7828-e4b3-4cb9-97f8-7efb5fa045b9.png)

**for booleans**
Booleans are correctly parsed by qs parse
![image](https://user-images.githubusercontent.com/85543813/233104177-f86b9638-931a-4ab6-b44d-0af2b6731e9e.png)